File: code-coaches/easy/zip-code-validator/ZipCodeValidator.cs
using System;

public class ZipCodeValidator
{
    static bool IsZipCodeValid(string zipCode)
    {
        try
        {
            int zipCodeInt = int.Parse(zipCode);
            return zipCode.Length == 5;
        }
        catch (FormatException)
        {
            return false;
        }
    }

    static void Main() {
        string zipCode = Console.ReadLine();

        if (IsZipCodeValid(zipCode))
            Console.WriteLine("true");
        else
            Console.WriteLine("false");
    }
}

File: code-coaches/easy/zip-code-validator/ZipCodeValidator.swift
func isZipCodeValid(_ zipCode: String) -> Bool {
    var isValid = false
    if let zipCodeInt = Int(zipCode) {
        isValid = zipCode.count == 5
    }
    return isValid
}

func main() {
	print(isZipCodeValid(readLine()!))
}

main()
